frontend/src/components/employees/employees.module.scss

```diff
@@ -0,0 +1,50 @@
+.employees {
+  grid-area: main;
+  display: grid;
+  margin: 0 1em;
+  grid-template:
+    'header addbutton' 5em
+    'table table' auto
+    / auto 10em;
+}
+
+h1 {
+  grid-area: header;
+  font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
+}
+
+.addButton {
+  grid-area: addbutton;
+  margin: 2em;
+}
+
+Table {
+  grid-area: table;
+}
+
+.editbtn,
+.removebtn {
+  text-align: right;
+  width: 1em;
+}
+
+.editButton,
+.removeButton {
+  border: none;
+  outline: none;
+  background-color: transparent;
+  font-weight: 100;
+  margin: 0.15em;
+  font-size: 1em;
+}
+
+.removeButton {
+  color: red;
+  font-weight: 1000;
+}
+
+.editButton:hover,
+.removeButton:hover {
+  font-size: 1.2em;
+  margin: 0em;
+}
```

Comment:
I think you can put this inside the `.editButton, .removeButton` section on line 38, using the `&:hover{ // stuff here }`